docs/code-style-guide.md

```diff
@@ -103,6 +114,51 @@ If you think some ASCII art will help explain something better, go for it! [asci
   float power(float a, float b);
   ```
 
+### Curly Braces
+
+* Opening and closing braces \(`{}`\) must be on the same indentation level.
```

Comment:
This is enforced by clang-format, so we don't need this section. I also was going to remove it in #1012

Reply:
ok removed